### PR TITLE
Add support for finding RSS Feeds

### PR DESCRIPTION
**Pull request by addyosmani**
This change fixes support for using the findFeeds functionality to discover RSS feeds via the Google Feeds API.

### CODE REVIEW

File: google-feeds.html [hunk 1]
```diff
@@ -96,7 +96,7 @@
     queryChanged: function() {
       if (this.query) {
         this.loading = true;
-        this.findFeeds();
+        this.withFeedsApi(this.findFeeds.bind(this));
```
Comment (ebidel) [inline google-feeds.html:99]: you could set this as a private prop in `ready()`. That way it's not bound every query change.

`this._feedHandler = this.findFeeds.bind(this);`

File: demo.html [hunk 2]
```diff
@@ -28,7 +31,11 @@
       console.log(e.detail.feed);
     });
 
-    var feeder = document.getElementById("feeder");
+    window.addEventListener('google-feeds-queryresponse', function(e) {
```
Comment (ebidel) [inline demo.html:34]: maybe a shorter event name: `google-feeds-response`

**Comment by ebidel** [comment]
Reviewed

**Comment by addyosmani** [comment]
This PR is no longer required as the query/findFeeds functionality mostly already landed in other PRs. Just one last fix in https://github.com/GoogleWebComponents/google-feeds/commit/7d0c4e013714b05e8eef058a6ec82e13d7936086

**Pull request closed without merging**